scripts/connect_clouddb_proxy.py

```diff
@@ -16,16 +16,22 @@
 from pocs.utils.config import load_config
 
 
-def main(instance_id, local_port, verbose=False):
+def main(instances, key_file, verbose=False):
```

Comment:
FWIW, there are inconsistent file names involved here: cloud_sql_proxy and connect_clouddb_proxy.py (i.e. _sql vs. db).

Reply:
That was somewhat intentional since I think users might not know what "cloud_sql" refers to but could figure out "cloud_db". I'll make consistent though.